test/e2e/xsite/xsite_test.go

```diff
@@ -467,6 +468,87 @@ func TestSuspectAndHearbeatConfig(t *testing.T) {
 	expectHeartBeatConfiguration(t, testKubes["xsite2"], true, hbInterval, hbTimeout)
 }
 
+func TestMultipleSites(t *testing.T) {
+	// single cluster cross-site is enough.
+	testName := tutils.TestName(t)
+	clientConfig := clientcmd.GetConfigFromFileOrDie(kube.FindKubeConfig())
+	kube := tutils.NewTestKubernetes(clientConfig.CurrentContext)
+	namespace := fmt.Sprintf("%s-xsite2", tutils.Namespace)
+
+	defer kube.CleanNamespaceAndLogOnPanic(t, namespace)
+
+	infinispan := &ispnv1.Infinispan{
```

Comment:
We can reduce the verbosity a fair bit here by using `tutils.DefaultSpec(t, testKube, func(i *ispnv1.Infinispan) {` to define the CR.

Reply:
I had to set `RUN_LOCAL_OPERATOR=false` locally to make it work with that spec. Let's see if I need to update the GHA too.

Reply:
Added to the GHA too. It was failing without it.

Reply:
I think that's because with the `DefaultSpec` function and `RUN_LOCAL_OPERATOR` we're apply the webhook defaults. I'm surprised setting `!RUN_LOCAL_OPERATOR` works though, as that should prevent the Operator from being started so I wouldn't expect any reconciliation in the test :thinking:

Reply:
Am I doing something wrong? The cross-site tests deploy the operator in the `kind` clusters: 
https://github.com/infinispan/infinispan-operator/blob/afdfd22dbb978e4590639115169e765414187f01/scripts/ci/configure-xsite.sh#L43

Reply:
Nope, you're doing exactly the right thing ... I have just forgot that detail :smile: